src/components/utils/rhf-inputs/ag-grid-table-rhf/custom-ag-grid-table.js

```diff
@@ -69,6 +69,7 @@ export const CustomAgGridTable = ({
     name,
     columnDefs,
     defaultRowData,
+    defaultEmptyRowsNumber,
```

Comment:
I'm not sure if it's a behaviour we should keep

Reply:
What do you suggests instead ?